tests/test_tabular_bagel.py

```diff
@@ -304,3 +307,33 @@ def test_get_completed_participants_sessions(
             session_id=session_id,
         )
     ] == expected
+
+
+@pytest.mark.parametrize(
+    "fpath",
+    [
+        DPATH_TEST_DATA / "bagel1.tsv",
+        DPATH_TEST_DATA / "bagel2.tsv",
+        Path(__file__).parent
+        / ".."
+        / "docs"
+        / "source"
+        / "user_guide"
+        / "inserts"
+        / "mriqc_bagel.tsv",
+    ],
+)
+def test_load(fpath):
+    assert isinstance(Bagel.load(fpath), Bagel)
```

Comment:
This seems to be the only time you test a happy-path via `Bagel.load()`. I don't remember how an uncaught exception inside of an assert looks like in pytest, but might be cleaner to split the loading and the assert